tests/test_emnlp_conference.py

```diff
@@ -557,4 +560,195 @@ def test_submit_papers(self, test_client, client, openreview_client, helpers):
             signatures=['~Program_EMNLPChair1'],
             writers=[]
         ))
-        assert revision_stage_note
\ No newline at end of file
+        assert revision_stage_note
+
+    def test_desk_rejection_by_SAC(self, test_client, client, openreview_client, helpers):
+
+        #update desk-rejection invitation
+        openreview_client.post_invitation_edit(
+            invitations='EMNLP/2023/Conference/-/Edit',
+            readers=['EMNLP/2023/Conference'],
+            writers=['EMNLP/2023/Conferencee'],
+            signatures=['EMNLP/2023/Conference'],
+            invitation=openreview.api.Invitation(
+                id='EMNLP/2023/Conference/-/Desk_Rejection',
+                edit={
+                    'invitation': {
+                        'invitees': [
+                            'EMNLP/2023/Conference/Program_Chairs',
+                            'EMNLP/2023/Conference/Submission${3/content/noteNumber/value}/Senior_Area_Chairs',
+                            'EMNLP/2023/Conference/Submission${3/content/noteNumber/value}/Area_Chairs'
+                            ],
+                        'edit': {
+                            'signatures': {
+                                'param': {
+                                    'regex': 'EMNLP/2023/Conference/Submission${5/content/noteNumber/value}/Senior_Area_Chairs|EMNLP/2023/Conference/Submission${5/content/noteNumber/value}/Area_Chair_.*|EMNLP/2023/Conference/Program_Chairs'
+                                }
+                            },
+                            'note': {
```

Comment:
Maybe I should just remove the title from the content in the venue class. 

Also, not sure if it is worth it to add this option of allowing SACs and ACs to desk-reject to the venue request.

Reply:
I think it is a good idea to add a field in the request form asking who can desk reject papers and by the default is just the PCs.